addons/binding/org.openhab.binding.heos/src/main/java/org/openhab/binding/heos/internal/handler/HeosChannelHandlerShuffleMode.java

```diff
@@ -0,0 +1,63 @@
+/**
+ * Copyright (c) 2010-2018 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.heos.internal.handler;
+
+import org.eclipse.smarthome.core.library.types.OnOffType;
+import org.openhab.binding.heos.handler.HeosBridgeHandler;
+import org.openhab.binding.heos.internal.api.HeosFacade;
+import org.openhab.binding.heos.internal.resources.HeosConstants;
+
+/**
+ * The {@link HeosChannelHandlerShuffleMode} handles the SchuffelModechannel command
+ * from the implementing thing.
+ *
+ * @author Johannes Einig - Initial contribution
+ *
+ */
+public class HeosChannelHandlerShuffleMode extends HeosChannelHandler {
+
+    public HeosChannelHandlerShuffleMode(HeosBridgeHandler bridge, HeosFacade api) {
+        super(bridge, api);
+    }
+
+    /*
```

Comment:
Please leave out those @see docblocks, then leave it empty

Reply:
You mean remove all the blocks in all handler classes?

Reply:
Yes, rather left out then just containing a see , the `Override` already signals that it has a parent method

Reply:
Okay thanks.